api/conversion/conversion.go

```diff
@@ -0,0 +1,191 @@
+/*
+Copyright 2017 caicloud authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package conversion
+
+import (
+	"fmt"
+
+	"github.com/caicloud/cyclone/api"
+	newapi "github.com/caicloud/cyclone/pkg/api"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	// AdminUserID represents the default user id of service.
+	AdminUserID string = "admin"
+
+	// AdminUsername represents the default username of service.
+	AdminUsername string = "admin"
+)
+
+// ConvertPipelineToService converts the pipeline to service, as the running of pipeline dependents on service. This
+// is just a workaround, the pipeline will can be run directly in the future.
+func ConvertPipelineToService(pipeline *newapi.Pipeline) (*api.Service, error) {
```

Comment:
Are these temporary codes？Do you plan to use `pipeline` instead of `service` next month?